src/main/java/org/wso2/gw/emulator/Tester.java

```diff
@@ -22,64 +22,307 @@
 
 import io.netty.handler.codec.http.HttpMethod;
 import io.netty.handler.codec.http.HttpResponseStatus;
-import org.wso2.gw.emulator.core.Emulator;
-import org.wso2.gw.emulator.sampletcp.dsl.IncomingMessage;
-import org.wso2.gw.emulator.sampletcp.dsl.OutgoingMessage;
+import org.wso2.gw.emulator.dsl.Emulator;
+import org.wso2.gw.emulator.dsl.Operation;
+import org.wso2.gw.emulator.dsl.QueryParameterOperation;
+import org.wso2.gw.emulator.http.client.contexts.HttpClientConfigBuilderContext;
+import org.wso2.gw.emulator.http.client.contexts.HttpClientRequestBuilderContext;
+import org.wso2.gw.emulator.http.client.contexts.HttpClientResponseBuilderContext;
+import org.wso2.gw.emulator.http.client.contexts.HttpClientResponseProcessorContext;
+import org.wso2.gw.emulator.http.params.Header;
+import org.wso2.gw.emulator.http.params.QueryParameter;
+import org.wso2.gw.emulator.http.server.contexts.HttpServerOperationBuilderContext;
+
+import java.io.FileNotFoundException;
+
+import static org.wso2.gw.emulator.http.server.contexts.HttpServerRequestBuilderContext.request;
+import static org.wso2.gw.emulator.http.server.contexts.HttpServerResponseBuilderContext.response;
+import static org.wso2.gw.emulator.http.server.contexts.HttpServerConfigBuilderContext.configure;
 
-import static org.wso2.gw.emulator.http.dsl.dto.consumer.IncomingMessage.request;
-import static org.wso2.gw.emulator.http.dsl.dto.consumer.OutgoingMessage.response;
 
 public class Tester {
```

Comment:
Please add some meaningful name for the test class and categorize those classes based on the internal test methods( eg:- All path check related test cases are included for the same class)